src/lib/components/pill.svelte

```diff
@@ -22,15 +22,15 @@
 
 <style lang="postcss">
   .pill {
-    @apply flex items-center justify-center gap-1 rounded-full border-2 border-slate-100 px-3 py-1 text-sm;
+    @apply flex items-center justify-center gap-1 rounded-full border-2 border-slate-50 px-3 py-1  text-sm text-black text-inverse;
   }
 
   .gray {
-    @apply bg-badge text-slate-600;
+    @apply border-none bg-transparent text-primary;
   }
 
   .gray.active {
-    @apply surface-primary border-2  text-primary;
+    @apply surface-interactive;
```

Comment:
We're actually never using this class currently, so we can leave it as is for now

```suggestion
    @apply surface-primary border-2 text-primary;
```